src/python/pants/backend/python/lint/ruff/rules.py

```diff
@@ -64,7 +64,8 @@ async def run_ruff(
     )
 
     conf_args = [f"--config={ruff.config}"] if ruff.config else []
-    initial_args = ("--fix",) if request.is_fix else ()
+    # `--force-exclude` applies file excludes from config to files provided explicitly
+    initial_args = ("--force-exclude", "--fix") if request.is_fix else ("--force-exclude",)
```

Comment:
Sorry for the late suggestion, but I just thought of this: instead of repeating the flag twice, once in each if branch, we can do:

```suggestion
    initial_args = ("--force-exclude",) + (( "--fix",) if request.is_fix else ())
```

Reply:
nice, that didn’t come to my mind, thx!

I do prefer not repeating the flag - the additional brackets are a little downer so to speak but whatever